playbooks/edx-east/vpc_admin.yml

```diff
@@ -10,7 +10,6 @@
     - user
     - jenkins_admin
     - hotg
-    - alton
```

Comment:
Note: Check with DevOps to find out what this role does?  We had no variables set for it, and so couldn't run it.